ko/admin/scripts.rst

```diff
@@ -0,0 +1,106 @@
+
+:meta-keywords: scripts
+:meta-description: Various CUBRID scripts support CUBRID Managements
+
+***************
+cubrid 스크립트
+***************
+
+.. _cubrid_scripts_unloaddb_sh:
+
+unloaddb.sh 스크립트
+===================
+
+**unloaddb** 의 실행 시간을 단축하기 위해서 서로 다른 테이블을 언로드 하기 위한 **cubrid unloaddb** 명령어를 동시에 여러 개 실행할 수 있다. 
+**unloaddb.sh** 은 이러한 과정을 자동화해주는 shell script이다.
+
+다수의 자식 process에게 균등하게 unloaddb의 부하를 배분하여 실행시키기 위해서 unloadb.sh은 catalog에서 언로드 대상 테이블들의 크기 정보를 가져온다 (DBA 권한 필요). 
+
+테이블 크기를 기준으로 각 자식 프로세스들에게 언로드할 테이블들이 할당되며, 각 자식 프로세스들은 할당된 테이블들의 unloaddb를 동시에 수행한다.
+
+.. warning::
+
+	**제약사항**
+	  * Linux 환경
+	  * DBA 권한이 있는 사용자만 실행 가능
+
+
+다음은 **unloaddb.sh** 에 대한 [options]이다.
+
+.. program:: sh unloaddb.sh
+
+.. option:: -u user
+
+    언로딩할 데이터베이스의 사용자 계정을 지정한다. 옵션을 지정하지 않으면 기본값은 DBA가 된다. ::
+
+        sh unloaddb.sh -u dba demodb
+
+.. option:: -t no-process
+
+    동시에 실행될 자식 프로세스의 개수 (no-process, 기본: 8개, 최대 16개 ) ::
+
+        sh unloaddb.sh -4 demodb
```

Comment:
```suggestion
        sh unloaddb.sh -t 4 demodb
```

Reply:
Thank you for your comment, it was fixe !